src/middlewares/validatorSchemas.js

```diff
@@ -123,4 +123,22 @@ const validateProfile = [
     .withMessage('Avatar should not include spaces'),
 ];
 
-module.exports = { validateSignup, validateSignin, validateProfile };
+const validateAddEvent = [
+  body('avatar').not().isEmpty().withMessage('Avatar should not be empty'),
+  body('description')
+    .not()
+    .isEmpty()
+    .withMessage('Avatar should not be empty')
+    .isLength({ min: 120 })
+    .withMessage('Description must be at least 120 characters long'),
+  body('location').not().isEmpty().withMessage('Location should not be empty'),
+  body('launchDate').not().isEmpty().withMessage('Date should not be empty'),
```

Comment:
Let's validate `launchDate` too, it should be a validate date.

This might help, (haven't tested it yet, but should work)
```body('launchDate').isISO8601().toDate()```